src/test/java/uk/gov/hmcts/reform/et/syaapi/service/NotificationServiceTest.java

```diff
@@ -893,4 +905,71 @@ void sendNotResponseNotificationEmailToClaimantMissingEmail() throws Notificatio
             any()
         );
     }
+
+    @Test
+    void sendResponseToRequestYesNotificationEmailToClaimant() throws NotificationClientException {
```

Comment:
probably better to use @ParameterizedTest instead of two separate tests

Reply:
thanks, updated this to use @ParameterizedTest